### PR TITLE
#1271 rich text input for ASC

### PR DESCRIPTION
**Pull request by tomlovesgithub**
## What's included?
Swapped out the basic text input for our rich text input. 
Allowing admins to format this paragraph 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Update/create an exercises additional selection criteria

## Risk - how likely is this to impact other areas?
🟢 Low risk - This change relies upon the apply app suitably displaying the input text.
 So needs to be released alongside the accompanying apply changes. 

## Additional context
image

---
PREVIEW:PRODUCTION
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

**Comment by warrensearle** [comment]
>So needs to be released alongside the accompanying apply changes.
@tomlovesgithub can you include a link to the corresponding Apply changes

**Review by warrensearle — DISMISSED** [review]
This change also needs to be reflected on the Eligibility Information view page. Currently it shows the html code rather than rendering it
image

**Review by mbrookeswebdev — APPROVED** [review]

**Comment by KateMJAC** [comment]
This gets an approval from me. However I just need to sling this one to the User Testing Group and when I receive feedback i'll come back in to approve for deployment to production.

File: src/views/Exercises/Applications/Application.vue [hunk 1]
```diff
@@ -1879,6 +1879,11 @@
               >
                 <dt class="govuk-summary-list__key">
                   {{ item.title }}
+                  <!-- eslint-disable -->
+                  <!-- <p
+                    v-html="exercise.selectionCriteria[index].text"
+                  /> -->
+                  <!-- eslint-enable -->
```
Comment (warrensearle) [inline src/views/Exercises/Applications/Application.vue:1886]: Hey @tomlovesgithub do we need to keep this commented out code?

File: src/views/Exercises/Show/Eligibility.vue [hunk 2]
```diff
@@ -84,10 +84,12 @@
...
               </p>
-              <p>
-                {{ criterion.text }}
-              </p>
+              <!-- eslint-disable -->
+              <p
+                v-html="criterion.text"
+              />
             </li>
+          <!-- eslint-enable -->
```
Comment (warrensearle) [inline src/views/Exercises/Show/Eligibility.vue:92]: Warning, am being super picky now!
...be worth keeping the `eslint` comment pairs close together: the `eslint-enable` should really be one line up, so we are turning eslint back on as soon as we can, and so we don't lose it 🤓

**Review by HalcyonJAC — APPROVED** [review]
Very nice :)

image

**Review by KateMJAC — COMMENTED** [review]
After some patchy testing via UTG, I have sent this to the non-legal team who will be using this feature most. 

They will be adding feedback in this Slack channel by the end of the week, so if you need to deploy this in my abscence, please check there first. The team members testing are Sam Marzan-Esher and Lauren Yu.

**Comment by joy-ade** [comment]
Thank, Kate for the update.

**Review by HalcyonJAC — APPROVED** [review]
Seems to work fine - looks good to me.

**Review by warrensearle — APPROVED** [review]

**Comment by joy-ade** [comment]
@tomlovesgithub This change also needs to be reflected on the Eligibility Information view page. Currently it shows the html code rather than rendering itScreenshot (8)

**Comment by joy-ade** [comment]
@KateMJAC this was tested today and its worked as it should.

**Comment by joy-ade** [comment]
@tomlovesgithub @KateMJAC UTG have approved however we are waiting for Tom to finish the outstanding work he’s identified before giving final approval.

**Comment by KateMJAC** [comment]
Sent to UTG today for re-testing after changes.

**Comment by joy-ade** [comment]
Approved by UTG

**Review by joy-ade — APPROVED** [review]
Approved by UTG

**Pull request merged**